Packs/MicrosoftExchangeOnline/Integrations/EWSO365/README.md

```diff
@@ -1,135 +1,167 @@
-Notice that it might be required to set the ``Timeout`` field to a higher value.
+**IMPORTANT**:  
+`First fetch timestamp` field is used to determine how much time back to fetch incidents from. The default value is the previous 10 minutes, Meaning: If this is the first time emails are fetched from the destination folder, all emails from 10 minutes prior to the instance configuration and up to the current time will be fetched.
+When set to get a long period of time, the `Timeout` field might need to be set to a higher value.
+
 Pay special attention to the following fields in the instance settings:
 
 `Email Address` – mailbox to fetch incidents from.  
-`Name of the folder from which to fetch incidents` – use this field to configure the destination folder from where emails should be fetched. The default is Inbox folder. Please note, if Exchange is configured with an international flavor `Inbox` will be named according to the configured language.
+`Name of the folder from which to fetch incidents` – use this field to configure the destination folder from where emails should be fetched. The default is Inbox folder.
 
-## Commands
+#### Permissions
 
-You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook. After you successfully execute a command, a DBot message appears in the War Room with the command details.
+Impersonation rights required. In order to perform actions on the target mailbox of other users, the service account must be part of the _ApplicationImpersonation_ role.
+
+#### Limitations
+
+If Exchange is configured with an international flavor `Inbox` will be named according to the configured language.
+
+## Commands
 
-1.  Get the attachments of an item: ews-get-attachment
-2.  Delete the attachments of an item: ews-delete-attachment
-3.  Get a list of searchable mailboxes: ews-get-searchable-mailboxes
-4.  Move an item to a different folder: ews-move-item
-5.  Delete an item from a mailbox: ews-delete-items
-6.  Search a single mailbox: ews-search-mailbox
-7.  Get the contacts for a mailbox: ews-get-contacts
-8.  Get the out-of-office status for a mailbox: ews-get-out-of-office
-9.  Recover soft-deleted messages: ews-recover-messages
-10.  Create a folder: ews-create-folder
-11.  Mark an item as junk: ews-mark-item-as-junk
-12.  Search for folders: ews-find-folders
-13.  Get items of a folder: ews-get-items-from-folder
-14.  Get items: ews-get-items
-15.  Move an item to a different mailbox: ews-move-item-between-mailboxes
-16.  Get a folder: ews-get-folder
-17.  Expand a distribution list: ews-expand-group
-18.  Mark items as read: ews-mark-items-as-read
-19.  Send an email: send-mail
-20.  Retrieve item as eml: ews-get-items-as-eml 
-21.  Reply to an email: reply-mail
+<details>
+<summary><h3 style={{display: 'inline'}}>ews-get-attachment</h3></summary>
 
-### 1\. Get the attachments of an item
+### ews-get-attachment
 
-* * *
+#### Description
```

Comment:
Do we need the Description heading?  I would prefer if the text was directly under the command name.